libraries/chain/include/eosio/chain/wasm_interface_private.hpp

```diff
@@ -177,20 +159,8 @@ namespace eosio { namespace chain {
             std::vector<U8> bytes = {
                 (const U8*)codeobject->code.data(),
                 (const U8*)codeobject->code.data() + codeobject->code.size()};
```

Comment:
I think we can get rid of `bytes` too? could be up to 1MB, on the big side for a needless copy

Reply:
You are right. `bytes` is not needed any more. Removing it saves an unnecessary copy.